dask_ml/datasets.py

```diff
@@ -377,3 +379,84 @@ def make_classification(
     y = y.astype(int)
 
     return X, y
+
+
+def random_date(start, end):
+    delta = end - start
+    int_delta = (delta.days * 24 * 60 * 60) + delta.seconds
+    random_second = np.random.randint(int_delta)
+    return start + timedelta(seconds=random_second)
+
+
+def make_classification_df(
+    n_samples=10000,
+    response_rate=0.5,
+    predictability=0.1,
+    random_state=None,
+    chunks=None,
+    dates=None,
+    **kwargs,
+):
+    """
+    Uses the make_classification function to create a dask
+    dataframe for testing.
+
+    Parameters
+    ----------
+    n_samples : int, default is 10000
+        number of observations to be generated
+    response_rate : float between 0.0 and 0.5, default is 0.5
+        percentage of sample to be response records max is 0.5
+    predictability : float between 0.0 and 1.0, default is 0.1
+        how hard is the response to predict (1.0 being easist)
+    random_state : int, default is None
+        seed for reproducability purposes
+    chunks : int
+        How to chunk the array. Must be one of the following forms:
+        -   A blocksize like 1000.
+    dates : tuple, optional, default is None
+        tuple of start and end date objects to use for generating
+        random dates in the date column
+    **kwargs
+        Other keyword arguments to pass to `sklearn.datasets.make_classification`
+
+    Returns
+    -------
+    dask.DataFrame
```

Comment:
I have a slight preference for returning a tuple of (DataFrame, Series) here.

The Series will have the same index as the DataFrame. But we return `y` separately, rather than as a column in the DataFrame. This makes things a bit more consistent with the other dataset generators, which returns tuples for supervised problems.

Reply:
I was wondering what was standard here. Thanks!